lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -28,6 +37,16 @@ def stub_publishing_api_discard_draft(content_id)
         stub_request(:post, url).to_return(status: 200, headers: {"Content-Type" => "application/json; charset=utf-8"})
       end
 
+      def stub_publishing_api_publish_with_version(content_id, version, update_type = 'major')
+        stub_publishing_api_publish(content_id, {update_type: update_type, previous_version: version}.to_json)
+      end
+
+      def stub_any_publishing_api_put_content_publish_and_put_links(content_id)
```

Comment:
I don't understand why this should be different from `stub_publishing_api_put_content_links_and_publish`.